Makefile

```diff
@@ -5,9 +5,8 @@ GOLINT = golint
 GOTEST = $(GO) test --cover --race -v
 GOVET = $(GO) vet
 GO_FILES = $(wildcard *.go)
-GO_PACKAGES = storage syslogish weblog
+GO_PACKAGES = storage log weblog
```

Comment:
Oh. Actually, that is easier said than done. `gofmt` does not like how `glide nv` returns the packages, i.e., like this:

```
$ glide nv
./log/...
./storage/...
./tests/...
./weblog/...
.
```